addons/pos_sale/report/sale_report.py

```diff
@@ -61,12 +61,22 @@ def _select_pos(self):
             NULL AS order_id"""
 
         additional_fields_info = self._select_additional_fields()
+        additional_fields_info = {x: 'NULL' for x in additional_fields_info}
+        additional_fields_info = self._fill_pos_fields(additional_fields_info)
         template = """,
-            NULL AS %s"""
-        for fname in additional_fields_info.keys():
-            select_ += template % fname
+            %s AS %s"""
+        for value, fname in additional_fields_info.items():
+            select_ += template % (fname, value)
```

Comment:
```suggestion
        for fname, value in additional_fields_info.items():
            select_ += template % (value, fname)
```
the field name is the key, not the value in the additional_fields_info dict